### PR TITLE
Remove arrow for diagrams filter, modify, sort

### PR DESCRIPTION
**Pull request by pygmychameleon**

### CODE REVIEW

File: docs/diagrams/ModifyCommandExecute.puml [hunk 1]
```diff
@@ -16,7 +16,7 @@ package "Command" {
     class ModifyCommand as "ModifyCommand"
 }
 
-InternshipLogicManager -down-> ModifyCommand : executes
-ModifyCommand -right-> InternshipStorage : checks for duplicates\nand edits original copy if unique
+InternshipLogicManager -down- ModifyCommand : executes
```
Comment (javinchua) [inline docs/diagrams/ModifyCommandExecute.puml]: add arrow

**Review by javinchua — CHANGES_REQUESTED** [review]

File: docs/diagrams/ModifyCommandStore.puml [hunk 2]
```diff
@@ -13,12 +13,12 @@ package "Internship Book" {
...
 
-InternshipStorage -[hidden]right-> InternshipEntry
+InternshipStorage -[hidden]right- InternshipEntry
 
 hide InternshipEntry
 
 class StorageAction as "Storage Action: Original Entry With Edited Fields"
 
-StorageAction -up-> InternshipStorage : overrides and replaces original entry
+StorageAction -up- InternshipStorage : overrides and replaces original entry
```
Comment (javinchua) [inline docs/diagrams/ModifyCommandStore.puml]: add arrow

**Review by javinchua — CHANGES_REQUESTED** [review]

**Review by javinchua — APPROVED** [review]
LGTM

File: docs/diagrams/ModifyCommandExecute.puml [hunk 3]
```diff
@@ -16,7 +15,7 @@ package "Command" {
     class ModifyCommand as "ModifyCommand"
 }
 
-InternshipLogicManager -down-> ModifyCommand : executes
-ModifyCommand -right-> InternshipStorage : checks for duplicates\nand edits original copy if unique
+InternshipLogicManager -down- ModifyCommand : > executes
+ModifyCommand -up- InternshipStorage : < checks for duplicates\nand replaces original copy with edited one if unique
```
Comment (s-kybound) [inline docs/diagrams/ModifyCommandExecute.puml:19]: should the arrow here be the other way?

**Review by s-kybound — CHANGES_REQUESTED** [review]
Minor changes required

**Pull request merged**